A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -3102,7 +3110,7 @@ class FMCMainDisplay extends BaseAirliners {
      * Called after TOPerf, Flaps or THS change
      */
     tryCheckToData() {
-        if (isFinite(this.v1Speed) || isFinite(this.vRSpeed) || isFinite(this.v2Speed)) {
+        if (isFinite(this.v1Speed) || isFinite(this.vRSpeed) || isFinite(this.v2Speed) || isFinite(this.perfTOTemp)) {
```

Comment:
I am not sure whether this is correct or not, since V1, VR, V2 are the main condition on whether or not to display the message.